### DISCUSSION TITLE
Is there right click functionality in Vimium?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [philc/vimium#4351] [General]
I came across #1948 but unable to comprehend it how does this work as it involved too many technical terms that i have no idea.
The key binding `gm` doesnt have any mapping in my `?` list.

**Comment by philc** [comment]
Hey @bheeshmpita, the functionality in #1948 was prototyped, but never implemented in mainline Vimium unfortunately. At the time it was being worked on, there was doubt about whether it was a useful-enough feature, as implemented.

**Reply by bheeshmpita** [reply]
Thanks @philc, for responding. 
Currently, is there any similar functionality in vimium where i can get context menu as my use case involves right-clicks.

**Reply by philc** [reply]
Hey @bheeshmpita, there isn't at the moment.

**Comment by sound-friend** [comment]
replying to this a year later to see if there's been any updates! :)
